contributors/devel/sig-cli/kubectl-conventions.md

```diff
@@ -372,7 +372,7 @@ and as noted in [command conventions](#command-conventions), ideally that logic
 should exist server-side so any client could take advantage of it. Notice that
 this is not a mandatory structure and not every command is implemented this way,
 but this is a nice convention so try to be compliant with it. As an example,
-have a look at how [kubectl logs](https://git.k8s.io/kubernetes/pkg/kubectl/cmd/logs/logs.go) is implemented.
+have a look at how [kubectl logs](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go) is implemented.
```

Comment:
Can you please pin this to a commit so that the link does not go stale again?

Reply:
Thanks @palnabarun for the review and suggestion, could you please explain a little more that how can I do this

Reply:
You click on `Copy Permalink` on the top right menu on GitHub UI when you open a file in the browser.

Reply:
Ohk got it, you are asking to use permalink,,,sorry didn't understand it at first.

Reply:
Sure, I will update this